controllers/hibp.js

```diff
@@ -73,10 +75,10 @@ async function notify (req, res) {
           email,
           supportedLocales,
           date: HBSHelpers.prettyDate(supportedLocales, new Date()),
-          unsubscribeUrl,
           breachAlert,
           SERVER_URL: req.app.locals.SERVER_URL,
-          buttonValue: LocaleUtils.fluentFormat(supportedLocales, "report-scan-another-email"),
+          scanAnotherEmailHref: scanAnotherEmailHref,
+          unsubscribeUrl: EmailUtils.getUnsubscribeUrl(subscriber, utmID),
```

Comment:
There's some other kind of future refactor in here ... something where we put all URL-generators together and make them available with a handlebars helper function or something. But doesn't need to happen now.